atom/browser/api/atom_api_notification.cc

```diff
@@ -207,7 +217,9 @@ void Notification::BuildPrototype(v8::Isolate* isolate,
       .SetProperty("hasReply", &Notification::GetHasReply,
                    &Notification::SetHasReply)
       .SetProperty("actions", &Notification::GetActions,
-                   &Notification::SetActions);
+                   &Notification::SetActions)
+      .SetProperty("soundName", &Notification::GetSoundName,
```

Comment:
Having the cpp variable as `sound_` but the methods as `soundName` is a bit inconsistent, should probably choosing one and stick to it on both JS and cpp side of things